adapters/vrtcal/vrtcal.go

```diff
@@ -87,3 +92,26 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter) (adapters
 	}
 	return bidder, nil
 }
+
+func getReturnTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType, error) {
+	for _, imp := range imps {
+		if imp.ID == impID {
+			if imp.Banner != nil {
```

Comment:
This is actually considered as an anti-pattern. Consider a request with multiple formats, that is `video` and `banner` both set. Is your bidder service confirmed to always return a `banner` bid? If not necessarily then you might be assigning the wrong bid type here and that will cause the ad to not be properly rendered. Does your bidder service perhaps communicate the bid type in the bid extension? If so, we highly recommend you to use that to set the bid type.